### PR TITLE
Minor performance enhancement

### PR DESCRIPTION
**Pull request by jnasoy**
Hi @aeadedoyin , here's my revised PR without the formatting diff

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <link rel="icon" type="image/png" sizes="32x32" href="images/favicon.ico/favicon-32x32.png">
     <link rel="icon" type="image/png" sizes="96x96" href="images/favicon.ico/favicon-96x96.png">
     <link rel="icon" type="image/png" sizes="16x16" href="images/favicon.ico/favicon-16x16.png">
-    <link rel="manifest" href="images/favicon.ico/manifest.json">
+    <link rel="manifest" href="/manifest.json">
```
Comment (aeadedoyin) [inline index.html]: It's good that you have separated the manifest.json as it isn't explicitly tied to the favicons
How about changing the directory name from **favicon.ico** to **favicons** and effect changes across all references.

File: manifest.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (aeadedoyin) [inline manifest.json]: This is cool 🚀.

Though I noticed this includes only x192 and x512 android icon sizes (these are new, cool 🚀) do include the rest of the icon sizes available in this -> **images/favicon.ico/manifest.json** also the density attributes.
NB: Consistency is key in JSON structure: src -> sizes -> type -> density

File: js/imageLoad.js [hunk 3]
```diff
@@ -65,7 +65,7 @@ function createLogos(logoArray) {
...
+        const html = groupanchor + `<div class='logo'> <div class='logo__holder'> <div class='logo__image'> <img loading="lazy" src='logos/${logo.filename}/${logo.filename}.svg' alt='${logo.title}logo'> </div><div class='logo__download'> <div class='logo__download__overlay'> <a href='logos/${logo.filename}/${logo.filename}.svg' download='${logo.title} Logo.svg'> <span class='logo__download__overlay--svg'> Download SVG </span> </a> <a href='logos/${logo.filename}/${logo.filename}.png' download='${logo.title} Logo.png'> <span class='logo__download__overlay--png'> Download PNG </span> </a> </div></div></div><div class='logo__text'> <p class='logo__text--primary'><a href='${logo.url}' title='${logo.urlTitle}' target='_blank' class='logo__text--link'>${logo.title}</a></p><p class='logo__text--secondary'>${(logo.category+"").replace(","," / ")}</p></div></div>`;
```
Comment (aeadedoyin) [inline js/imageLoad.js]: Neat!

NB: Consistency is key single quote at `<img loading='lazy'` ... not double i.e  `<img loading="lazy"`...

File: manifest.json (hunk 2 above)
Comment (jnasoy) [reply manifest.json]: I think there is no need to add the density attribute.
See here https://github.com/RealFaviconGenerator/realfavicongenerator/issues/228

**Comment by jnasoy** [comment]
> Nice work 👍 @jnasoy @aeadedoyin
> I thought the addition of the service worker would make the site available offline but that doesn't seem to be the case here, I can now add the site to my home screen but offline availability isn't present. Is this intentional @jnasoy ?

Hi @demilad , that's one of the features of the service worker. I think the offline page feature is more on the accessibility category. Since I'm focusing on performance in this PR, I only added the cache feature to improve the speed of serving the assets.

But I can implement that feature and include it in this PR if you want. :)

**Comment by demilad** [comment]
> > Nice work 👍 @jnasoy @aeadedoyin
> > I thought the addition of the service worker would make the site available offline but that doesn't seem to be the case here, I can now add the site to my home screen but offline availability isn't present. Is this intentional @jnasoy ?
> 
> Hi @demilad , that's one of the features of the service worker. I think the offline page feature is more on the accessibility category. Since I'm focusing on performance in this PR, I only added the cache feature to improve the speed of serving the assets.
> 
> But I can implement that feature and include it in this PR if you want. :)

Thanks for the clarity @jnasoy 
I think any additional features can be a separate PR 👍🏽

**Pull request merged**